dags.yaml

```diff
@@ -1544,7 +1544,7 @@ bqetl_acoustic_suppression_list:
     - impact/tier_3
 
 bqetl_braze:
-  schedule_interval: 0 10 * * *
+  schedule_interval: 30 3,11,19 * * *
```

Comment:
this may change depending on what the final ctms times are set to and how long their jobs take to run